### DISCUSSION TITLE
`php artisan responsecache:clear` Command Does Not Works

### DISCUSSION THREAD

**Discussion by erhanask** [spatie/laravel-responsecache#404] [Bugs]
public function configurePackage(Package $package): void
    {
        $package
            ->name('laravel-responsecache')
            ->hasConfigFile()
            ->hasCommands([
                ClearCommand::class,
            ]);
    }
   

This method in ResponseCacheServiceProvider returns package with a command property. 
While codes like this `php artisan responsecache:clear` command does not works.

But if I set clear command like ,

$this->commands([
      ClearCommand::class,
 ]);
 
 it works fine. 
 
 I hope you can see the problem, btw I'm using v6.6.9 of the package.
 Thanks for your help from now.

**Comment by erhanask** [comment]
@freekmurze @rubenvanassche @willemvb @sebastiandedeyne @riasvdv @AlexVanderbist @jefvdv  Can anybody help me ?

**Reply by freekmurze** [reply]
Please do not mention our entire team. Currently, we don't have time to assist you with this problem.

Should you find that the package is in error, submit a PR.

**Reply by erhanask** [reply]
Sorry about that, I just thinked nobody noticed. Thanks.
